odoo/sql_db.py

```diff
@@ -608,7 +530,7 @@ def close(self):
             self._lock.release()
 
     def autocommit(self, on):
-        _logger.debug("TestCursor.autocommit(%r) does nothing", on)
+        warnings.warn("Deprecated method and does nothing")
```

Comment:
As discussed yesterday, add version so it's easier to remember when to remove it (or realise that we can remove it)

Reply:
Yes, you are perfectly right.